client/components/tinymce/plugins/wpcom-view/views/simple-payments/index.jsx

```diff
@@ -16,34 +17,36 @@ import { getSimplePayments } from 'state/selectors';
 import { getSelectedSiteId } from 'state/ui/selectors';
 import formatCurrency from 'lib/format-currency';
 import QuerySimplePayments from 'components/data/query-simple-payments';
+import QueryMedia from 'components/data/query-media';
+import { getMediaItem } from 'state/selectors';
 
 class SimplePaymentsView extends Component {
 	render() {
-		const { productId, product, siteId } = this.props;
+		const { productId, product, siteId, productImage } = this.props;
```

Comment:
really minor comment. if `! product` is `true` You are going to declare `productImage` needlessly.